developer-docs-site/docs/guides/get-test-funds.md

```diff
@@ -0,0 +1,67 @@
+## Create and Fund Aptos Account
+
+For testing purposes, you will want to create an Aptos account and fund it with testnet tokens. This document accompanies the command line instructions for the Aptos CLI on [creating](../cli-tools/aptos-cli-tool/use-aptos-cli.md#initialize-local-configuration-and-create-an-account) and [funding]()(../cli-tools/aptos-cli-tool/use-aptos-cli.md#fund-an-account-with-the-faucet) accounts, focusing upon the testnet network and leveraging a graphical wallet rather than the CLI.
+
+## Prerequisites
+
+You will need these installed to proceed:
+
+* The wallet of your choice; we use the [Petra Wallet](../../guides/install-petra-wallet.md) Chrome extension.
+* [Aptos CLI](../cli-tools/aptos-cli-tool/index.md)
+
+## Create Aptos account
```

Comment:
This can be optional, because people could use the key directly from the wallet I think the important thing here is:
1. Create an address in your preferred wallet (or the CLI)
2. If you created it with the CLI, import the private key into the wallet
3. Switch to the `testnet` network.  Please consult your wallet's documentation for this information (we can show how to do in Petra)
4. The wallet connection through the application to get the testnet funds.

Reply:
I don't understand step 4. I've instead explained most of this can be done in the wallet. We already have docs on that at petra.app.